vuln/core/1.json

```diff
@@ -6,5 +6,6 @@
   "vulnerable": "8.x || 7.x || 4.x || 6.x || 5.x",
   "patched": "^8.1.4 || ^7.10.1 || ^4.8.4 || ^6.11.1",
   "description": "memory overread when parsing invalid NAPTR responses",
-  "overview": "The c-ares function ares_parse_naptr_reply(), which is used for parsing NAPTR\nresponses, could be triggered to read memory outside of the given input buffer\nif the passed in DNS response packet was crafted in a particular way.\n\n"
+  "overview": "The c-ares function ares_parse_naptr_reply(), which is used for parsing NAPTR\nresponses, could be triggered to read memory outside of the given input buffer\nif the passed in DNS response packet was crafted in a particular way.\n\n",
+  "affectedEnvironments": ["linux", "osx", "win"]
```

Comment:
I think we need a input for: 'all' or '*'

Reply:
Solved in https://github.com/nodejs/security-wg/pull/912/commits/e3b8f8ae41e11601a45d08238d24d3ecfdfb3502 and [d9e544a](https://github.com/nodejs/security-wg/pull/914/commits/d9e544a102bfcf462f96fcdfb5d62ed701b663f1)